containers/deploy/service-accounts/tasks/main.yml

```diff
@@ -31,10 +23,6 @@
       metadata:
         name: anyuid
         namespace: "{{ project_name }}"
-  tags:
-    - start
 
 - name: Allow anyuid service account to create secrets
   command: oc adm policy add-role-to-user edit -z anyuid
```

Comment:
This can be done with the Ansible module by creating a resource with the following structure:
```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  name: anyuid-secrets
  namespace: foreman
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: Role
  name: edit
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: ServiceAccount
  name: anyuid
```